runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapPoint.java

```diff
@@ -28,10 +28,12 @@
 import java.awt.event.MouseEvent;
 import java.awt.image.BufferedImage;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import net.runelite.api.Point;
 import net.runelite.api.coords.WorldPoint;
 
 @Data
+@EqualsAndHashCode(of = "worldPoint")
```

Comment:
Will this cause two WorldMapPoint's of the same location to return as equal? Not very familiar with @EqualsAndHashCode

Reply:
Yeah. Not sure if that is the default behaviour, but i just abused the HashSet to avoid checking if worldpoints are already set in the list or not.

Reply:
I can probably extend it to tooltip too, or idk what else should I consider there.